common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_apmanager.yaml

```diff
@@ -139,6 +139,9 @@ cACTION_APMANAGER_CLIENT_IRE_CONNECTED_NOTIFICATION:
   _type: class
   mac: sMacAddr
 
+cACTION_APMANAGER_ACK:
+  _type: class
```

Comment:
I think this should be added together with ErrorCodeTLV, for now there is no way even to fill those fields.

Reply:
Indeed, the STA check is the only thing that can possibly fail and that is not implemented yet (because the slave / ap manager doesn't have the station list).

#326 covers this.

But on the other hand, @morantr is right that to be able to implement that, we will need to add reason and sta_mac fields (or ErrorCodeTLV) to cACTION_APMANAGER_ACK.

But of course, if #435 gets implemented first, the point is moot because all the cACTION_APMANAGER can be removed.

Reply:
Done in #437